### PR TITLE
Conic gradient quick post

### PR DESCRIPTION
**Pull request by rachelandrew**
A quick post about conic gradients.

(also using as a demo of this type of post).

### CODE REVIEW

File: src/site/content/en/blog/conic-gradient-border/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Use conic gradients to create a cool border
+subhead: >
+  Conic gradients can be used to create some interesting effects, such as this nice border example. 
+date: 2022-04-11
+hero: image/kheDArv5csY6rvQUJDbWRscckLr1/U4PTWqvLRcQwgK4Lv9Pf.png
```
Comment (malchata) [inline src/site/content/en/blog/conic-gradient-border/index.md:6]:
```suggestion
hero: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/xK2kXKqAmfaofSEN5XJ3.jpg
```

Comment (malchata) [reply src/site/content/en/blog/conic-gradient-border/index.md:6]: Per our guidelines, hero images should be 3200x960.

**Review by malchata — APPROVED** [review]
Minor change to resize the hero image, otherwise LGTM!

File: src/site/content/en/blog/conic-gradient-border/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: Use conic gradients to create a cool border
+subhead: >
+  Conic gradients can be used to create some interesting effects, such as this nice border example. 
+date: 2022-04-11
```
Comment (rachelandrew) [inline src/site/content/en/blog/conic-gradient-border/index.md:5]:
```suggestion
date: 2022-05-27
```

**Pull request merged**